src/components/GrowthChart/GrowthChart.tsx

```diff
@@ -79,44 +79,41 @@ export const GrowthChart = ({
     };
 
     return (
-        <div>
-            <div className='flex flex-wrap-reverse pl-14'>
-                <div>
-                    <ChartSelector
-                        category={category}
-                        dataset={dataset}
-                        setCategory={setCategory}
-                        setDataset={setDataset}
-                        chartData={chartDataForGender}
-                        isDisabled={trackedEntityGender !== undefined}
-                        gender={gender}
-                        setGender={setGender}
-                    />
-                </div>
-                <div className='grow relative min-w-[100px] text-sm'>
+        <>
+            <div className='flex justify-between px-14'>
+                <ChartSelector
+                    category={category}
+                    dataset={dataset}
+                    setCategory={setCategory}
+                    setDataset={setDataset}
+                    chartData={chartDataForGender}
+                    isDisabled={trackedEntityGender !== undefined}
+                    gender={gender}
+                    setGender={setGender}
+                />
+                <div className='relativ'>
```

Comment:
"relativ" isn't a option with tailwindcss, either remove it or use "relative" as in the suggestion.
```suggestion
                <div className='relative'>
```